kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -3465,6 +3465,7 @@ components:
         - CONNECT
         - KSQL
         - ACL
+        - AUDIT
```

Comment:
Would you please update application-local.yml to include new RBC operation there as well?